From d3c233b6e37952f60e705bcd80a3b00fdc40e079 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 1 Mar 2025 23:19:40 -0500
Subject: [PATCH] Add system driver locations

---
 meson.build | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 9b71bf7..369ac6a 100644
--- a/meson.build
+++ b/meson.build
@@ -75,7 +75,13 @@ if driverdir == ''
   if host_machine.system() == 'windows'
     driverdir = join_paths(get_option('prefix'), get_option('bindir'))
   else
-    driverdir = join_paths(get_option('prefix'), get_option('libdir'), 'dri')
+    driverdir = '@0@/dri:@1@/dri:@2@/dri'.format(
+      join_paths(get_option('prefix'), get_option('libdir')),
+      # This is where the Ubuntu package installs the drivers
+      '/usr/lib/@MULTIARCH_PLACEHOLDER@',
+      # This seems to be where Fedora puts them
+      '/usr/lib64/'
+    )
   endif
 endif
 
-- 
2.48.1
